client/package.json

```diff
@@ -68,7 +68,7 @@
     "@babel/preset-env": "^7.7.6",
     "@babel/preset-react": "^7.7.4",
     "@rollup/plugin-eslint": "^8.0.1",
-    "@tacc/core-styles": "^0.5.1",
+    "@tacc/core-styles": "^0.5.2-alpha",
```

Comment:
To get more padding for small buttons:
1. https://github.com/TACC/Core-Styles/pull/30
2. https://github.com/TACC/Core-Styles/commit/eafe102

Reply:
> "@tacc/core-styles": "^0.5.2-alpha",

Anything preventing this from being a full patch release?

Reply:
No, but I have a reason (a potentially misguided one). Please evaluate.

<details><summary>Reason</summary>

I am trying out a new process to show what has and has not been tested by client software (Portal, CMS, Components). Because we don't have Storybook yet[^1], I can't reliably test Core-Styles changes except in client software (like isn this PR).

0. It was faster, because I didn't plan to log alpha/beta releases in changelog.
1. changes made for dependent PRs of client software get alpha/beta releases
2. full releases (major/minor/patch) are thus all those alpha/beta releases that got tested and merged
3. when core-styles can be tested via storybook, the alpha/beta release solution is dropped

Though this specific change is definitely reliable enough for a 0.5.2 release, it was only one change, so I thought I should wait till it had more changes.

</details>

[^1]: Joon Yee or I will implement Storybook after we have working Core-Components, which he is taking over for me, today.

Reply:
P.S. I'll release 0.5.2 today and update this. Gotta be later in afternoon, though.